sdk/monitor/azure-monitor-opentelemetry-exporter/swagger/README_QUICKPULSE.md

```diff
@@ -19,9 +19,9 @@ description: Quickpulse Client
 generated-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 no-namespace-folders: true
-output-folder: ../azure/monitor/opentelemetry/exporter/quickpulse/_generated
-source-code-folder-path: ./azure/monitor/opentelemetry/exporter/quickpulse/_generated
-input-file: https://quickpulsespecs.blob.core.windows.net/specs/swagger-v2-for%20sdk%20only.json
+output-folder: ../azure/monitor/opentelemetry/exporter/_quickpulse/_generated
+source-code-folder-path: ./azure/monitor/opentelemetry/exporter/_quickpulse/_generated
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/665e7c3b6f26b148b3c05e55602621bc293cc0a4/specification/applicationinsights/data-plane/LiveMetrics/preview/2024-04-01-preview/livemetrics.json
```

Comment:
Is this eventually going to move back to the original quickpulse swagger or no? I assumed the existing swagger was updated, but this looks like we're migrating to a new swagger json.

Reply:
We are migrating to a new swagger json. There are currently some issues with it that Benke is working on to fix (the two issues I outlined above) and we are manually hardcoding them to work with our sdk for now.